layouts/partials/events/render-event.html

```diff
@@ -72,11 +74,10 @@ <h2>Examples</h2>
     */}}
     {{ if $desired_example_name }}
         {{ if eq $example_name $desired_example_name }}
-            {{ $example_content := partial "json-schema/resolve-refs" (dict "schema" $example "path" "event-schemas/examples") }}
-```json
-{{ jsonify (dict "indent" "  ") $example_content }}
-```
+            {{ partial "events/example" (dict "example" $example)}}
         {{ end }}
+    {{ else if eq $event_name $example_name }}
+        {{ partial "events/example" (dict "example" $example)}}
     {{/*
       If `$desired_example_name` is not given, we will include any
       examples whose first part (before "$") matches the event name
```

Comment:
(From the description of the PR)

> (examples are still shown under the main type, but are now additionally shown under the subtype)

I'm far from convinced this is useful. Having 15 different examples under https://pr1283--matrix-spec-previews.netlify.app/client-server-api/#mroommessage doesn't seem to add much, particularly when the examples are all duplicated later. What if we just used the `m.text` example (via `desired_example_name`, presumably)?

Reply:
I tried to do this with https://github.com/matrix-org/matrix-spec/pull/1283/commits/d6345f0df2fec12c08aaadac6fe3c6423a4bf87b#diff-4bdc5bb1f13036061393a5799f5ec426a67bf2850c8083d21e0b649877ff6bc8L13 but `m.room.message` still has all its examples, so I seem to be missing something. :-/

Reply:
Figured it out.  This is one done now.